pkgs/sops-age-hook/sops-age-hook.bash

```diff
@@ -0,0 +1,25 @@
+_sopsAddKey() {
```

Comment:
I would suggest not to add a new sops-age-hook but instead encourage users to setup a .sops.yaml instead.

https://github.com/Mic92/dotfiles/blob/master/nixos/.sops.yaml

The reason is that `SOPS_AGE_RECIPIENTS` does not work with multiple secret files.

Reply:
> The reason is that `SOPS_AGE_RECIPIENTS` does not work with multiple secret files.
Can you clarify what you mean by this? `SOPS_AGE_RECIPIENTS` can encrypt to multiple public keys, I think I have it doing so on my local config.

I do think that we should encourage usage of a `.sops,yaml` though, it's nice and declarative :) I wasn't aware that it was even an option. Seems like i didn't read sops documentation thoroughly enough ...

Reply:
another issue with this hook if we were to keep it -- I don't think it handles multiple public keys in one file correctly, which age seems to encourage

Reply:
Ok. To clarify. It does not work if you are trying to have multiple sops files with different recipients, which you can do with the .sops.yaml approach.

Reply:
This should be removed now. See https://github.com/Mic92/sops-nix/issues/98